.github/workflows/publish-on-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish Cookie Policy release
+
+# Only for testing. Should be on: release: types: [published]
+on:
+  - push
```

Comment:
Not sure if we want that yet, but could make sense to simply publish on every merge to main branch?

This would not create a release tag on GH (unless we can make it automated as well), but may be a nice next step in automating this, instead of relying on manual release.

This is the way we have it set up for our python packages that publish to pip, and in react components I think.
In Vanilla we want more manual control, so we tie it to GH release functionality. But here it probably will make it easier for everyone if it's fully automatic.

Reply:
Like here: https://github.com/canonical/canonicalwebteam.flask-base/blob/main/.github/workflows/publish.yaml

Reply:
I agree on the automation, and there should a way to create the release tag. In fact, we could 
- create the tag
- create the release on GH
- publish to npm 

on merge to main. 

I think this works for now though, and I can make another PR to set up the merge to main actions.